fog/distribution/src/main.rs

```diff
@@ -318,13 +291,45 @@ fn main() {
     thread::sleep(Duration::from_secs(1));
 }
 
+fn rebuild_fogresolver(
```

Comment:
nit: this should be `build_fog_resolver`

and should include the line

```
    let report_verifier = {
        let mr_signer_verifier = fog_ingest_enclave_measurement::get_mr_signer_verifier(None);
        let mut verifier = Verifier::default();
        verifier.debug(DEBUG_ENCLAVE).mr_signer(mr_signer_verifier);
        verifier
    };
```
and the `FogResolver::new` part